integreat_cms/api_client_firebase/push_notification_sender.py

```diff
@@ -0,0 +1,156 @@
+
+import logging
+import requests
+
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+
+from ..cms.forms.push_notifications.push_notification_translation_form import PushNotificationTranslation
+from ..cms.constants import push_notifications as pnt_const
+from ..cms.models import Region
+logger = logging.getLogger(__name__)
+
+# Firebase Push Notifications / Firebase Cloud Messaging #
+class PushNotificationSender:
```

Comment:
```suggestion
class FirebaseApiClient:
```
I would rename this file to "firebase_api_client.py"